lib/converge/api.ts

```diff
@@ -108,3 +111,19 @@ export async function loadProvider(graphClient: GraphClient,
         options: QueryNoCacheOptions,
     })).SCMProvider[0];
 }
+
+export async function saveGitHubAppUserInstallations(
```

Comment:
Please align parameters with function ():

Reply:
@cdupuis done - though I would expect a linter/autofix to catch such things.